### DISCUSSION TITLE
transitions not working with styled function

### DISCUSSION THREAD

**Discussion by lucasvsouza28** [stitchesjs/stitches#963] [Help] [answered]
I've tried to create a simple progressbar component using the `styled` function.
The problem i'm facing is that the `transition` property are being setted, but the animation is not working.
I've changed the `styled` function to `css` function and the animation works properly with no changes in the component styles.

My question is: the `styled` function are equivalent to `css`? Should them behave the same?

## created with styled
![styled_not_animated](https://user-images.githubusercontent.com/23587471/155000683-5e56aa1d-da55-4a32-9b68-897f20b3b4fc.gif)

## created with css
![css_animated](https://user-images.githubusercontent.com/23587471/155000719-8b0999a3-17a0-400f-a5e7-ec83c64c3878.gif)

**Comment by hadihallak** [comment, marked as answer]
Please create an issue with what you're describing 🙏

**Reply by lucasvsouza28** [reply]
ok, thank you

**Reply by lucasvsouza28** [reply]
Issue #964 created
